src/zoid/buttons/component.jsx

```diff
@@ -714,6 +714,9 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
                 button_message_color: message?.color,
                 button_message_offer_country: offerCountryCode,
                 button_message_amount: amount,
+                button_message_currency: currency,
+                // comes back as an array
+                [FPTI_KEY.SELLER_ID]: merchantID[0],
```

Comment:
There can be more then one payee in a transaction. Do you want to log only one payee in fpti?

Reply:
good question, I am not sure if we need both. Is there an order of how this array is generated? I found a couple examples in scnw only logging the first one so i assumed that might be the more "important" one. Are you able to clarify that?